lib/high_five/thor/tasks/deploy.rb

```diff
@@ -169,6 +169,18 @@ def deploy(target)
             say "Cloning to #{@config.dev_index}"
             FileUtils.cp(File.join(self.destination_root, "index.html"), File.join(@config.root, @config.dev_index))
           end
+
+          if (@config.cordova_path)
+            cordova_path = File.join(base_config.root, @config.cordova_path)
+            pwd = Dir.pwd
+            Dir.chdir cordova_path
```

Comment:
you can pass a block to `chdir` and it will automatically set the path back after it exits